vacasa/connect/connect.py

```diff
@@ -1604,25 +1603,37 @@ def create_contact(self,
             Created Contact
 
         """
-
         payload = {
             "first_name": first_name,
-            "last_name": last_name,
-            "address_1": address_1,
-            "address_2": address_2,
-            "city": city,
-            "state": state,
-            "zip": zip,
-            "country_code": country_code,
             "email": email,
-            "phone": phone,
-            "phone_notes": phone_notes,
-            "language_id": language_id,
-            "created_by": created_by,
-            "tax_entity_name": tax_entity_name,
-            "send_email": send_email,
+            "last_name": last_name
         }
 
+        if address_1:
+            payload["address_1"] = address_1
+        if address_2:
+            payload["address_2"] = address_2
+        if city:
+            payload["city"] = city
+        if state:
+            payload["state"] = state
+        if zip:
+            payload["zip"] = zip
+        if country_code:
+            payload["country_code"] = country_code
+        if phone:
+            payload["phone"] = phone
+        if phone_notes:
+            payload["phone_notes"] = phone_notes
+        if language_id:
+            payload["language_id"] = language_id
+        if created_by:
+            payload["created_by"] = created_by
+        if tax_entity_name:
+            payload["tax_entity_name"] = tax_entity_name
+        if send_email:
+            payload["send_email"] = send_email
+
```

Comment:
are any of these nullable?  if not, something like this might work
```python
payload = {k: v for k, v in payload.items() if v is not None}
# or 
payload = {k: v for k, v in payload.items() if v} 

```

Reply:
I was going to do that originally, but I opted for readability?

Reply:
I think adding a comment along with the comprehension would be a nice balance.

Reply:
dict comprehension is a pretty standard python idiom, there's definitely a balance but i'd trust your original instinct in this case 😄